pkgs/by-name/ht/httpy-cli/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, python3Packages
+, fetchPypi
+ }:
+
+python3Packages.buildPythonPackage rec {
+  pname = "httpy-cli";
+  version = "1.1.0";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "httpy-cli";
+    hash = "sha256-uhF/jF4buHMDiXOuuqjskynioz4qVBevQhdcUbH+91Q=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    colorama
+    pygments
+    requests
+    urllib3
+  ];
+
+  meta = with lib; {
+    description = "Modern, user-friendly, programmable command-line HTTP client for the API";
+    homepage = "https://github.com/knid/httpy";
+    license = licenses.mit;
+    mainProgram = "httpy";
+    maintainers = with maintainers; [ eymeric ];
```

Comment:
Oh I just notice that the maintainer is not you, is it intended?

Reply:
it's me Eymeric (github account hatch01)

Reply:
Oh... sorry about that. The confusing part is that your Nix contributor pseudo is different from your Github account. All good anyway!